### PR TITLE
feat: added lists and slider

### PR DESCRIPTION
**Pull request by hiteshverma27**

### CODE REVIEW

File: Pages/lists.html [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+    <link rel="stylesheet" href="/Styles/OrcaUI-styles.css">
+    <link rel="stylesheet" href="/Styles/main.css">
+    <meta name="author" content="Hitesh Verma">
+    <meta name="Description" content="A variety of avatar to choose from.">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <script src="/JS/toggle.js" defer></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+    <link rel="shortcut icon" href="/Assests/logo-rotated.svg" type="image/x-icon">
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-T7RGFXZS8P"></script>
```
Comment (prakashsakari) [inline Pages/lists.html:18]: you can create a separate js file or add script in the body rather than the head

File: Pages/slider.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Clabel%2520for%253D%2522input%2522%253EPrice%2520range%253C%252Flabel%253E%250A%253Cinput%2520id%253D%2522input-slider%2522%2520type%253D%2522range%2522%2520min%253D%25220%2522%2520max%253D%252210000%2522%253E"
+  style="width: 511px; height: 95px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+            </div>
+            <div class="example-box flex-col my-4">
+                <label for="input">Price range</label>
+                <input id="input-slider" type="range" min="0" max="10000">
+            </div>
+            <hr>
```
Comment (prakashsakari) [inline Pages/slider.html:143]: just a suggestion you can also use a border-bottom instead of hr

**Review by prakashsakari — COMMENTED** [review]
Added some comments, the rest looks good to me

**Review by ishitakeshawani — APPROVED** [review]
LGTM

File: Pages/slider.html (hunk 2 above)
Comment (hiteshverma27) [reply Pages/slider.html:143]: Its kind of the theme of the website but thanks for the suggestion

File: Pages/lists.html (hunk 1 above)
Comment (hiteshverma27) [reply Pages/lists.html:18]: Yes will do

**Pull request merged**